### ISSUE TITLE
Can I set column header to automatically wrap if it's too long

### ISSUE THREAD

**Issue by hongyiqin** [quantopian/qgrid#107]
Currently if the text in column headers are too long, parts of the text are hidden. Is it possible to ask the header row to automatically adjust its height so the text can be wrapped and displayed in the next line, similar to how Jupyter Notebook displays the dataframe.

so basically, from:

This is a very long he | Second Header
------------ | -------------
value | value

to:

This is a very long <br> header | Second Header
------------ | -------------
value | value

Thanks.

**Comment by TimShawver** [comment]
I don't think this is possible without making some CSS changes, from what I'm reading here: https://github.com/mleibman/SlickGrid/issues/61

So unless you're up for hacking on some CSS in your local copy of qgrid I don't think it's currently possible.  I can look into adding it later possibly so thanks for bringing it to my attention.

**Comment by jaladh-singhal** [comment]
@TimShawver Is there any progress on this issue?

Or possibly a better way to change CSS externally instead of changing the internal CSS code of package, like is there a QgridWidget object's property or method which can allow to change CSS of headers? In grid options of slickgrid there is `headerRowHeight` but it is apparent that QgridWidget has a different way of generating header row because `showHeaderRow` is set to False - what am I missing here?
